chore: update claude sdk vs deepagents docs (#3596)
langchain-ai/docs: src/oss/deepagents/comparison.mdx

| **Security configurability** | Composable, per-tool human-in-the-loop | Permission system with modes, rules and hooks | Built-in tiers using approval modes and OS-level sandboxes |
| **License** | MIT | MIT (underlying Claude Code is proprietary) | Apache-2.0 |

### Summary

> - **Choose Deep Agents** for an open source, model-agnostic harness you can deploy in a production-ready, [multi-tenant](/oss/deepagents/going-to-production#multi-tenancy) way while [keeping control of your memory](https://x.com/hwchase17/article/2042978500567609738).

Reviewer: ```suggestion
- **Choose Deep Agents** for an open source, model-agnostic harness you can deploy in a production-ready, [multi-tenant](/oss/deepagents/going-to-production#multi-tenancy) way while [keeping control of your memory](https://www.langchain.com/blog/your-harness-your-memory).
```

- **Choose Claude Agent SDK or Codex SDK** if you want to standardize on a single model provider and are comfortable building the hosting, auth, and multi-tenant isolation layers yourself.

## Feature comparison

> Instead of one large matrix, the sections below group the comparison by decision area.

Reviewer: ```suggestion
The feature comparison is grouped by decision area.
```
